sale_product_multi_add/wizards/sale_import_products_view.xml

```diff
@@ -33,7 +33,7 @@
                 <footer>
                 <button
                         name="select_products"
-                        string="Select"
+                        string="Confirm"
```

Comment:
Why changing this ? Maybe a text help could be valuable in this wizard form. Something like 'Please select the products you want to add to the Sale Order, then click on 'Select' ?

Reply:
> Why changing this ? Maybe a text help could be valuable in this wizard form. Something like 'Please select the products you want to add to the Sale Order, then click on 'Select' ? @jbaudoux

I suggested to change to text from 'Select' to 'Confirm' as IMHO it makes more sense to be like so. As we are in the last step of dynamically adding various Sales Order Lines, we are not selecting anything, we are just confirming to add one/multiple order lines.

Then, a help text with the text you are suggesting would make sense on "wizard num 1", but not on the second one where you are just assigning the quantities for each product, and then confirming to add the products to the Sale Order. The text could be useful on "wizard num 1", nevertheless, I believe it is quite intuitive already that you need to add some products, but we can discuss on that of course.

What do you think?